cetmix_tower_server/models/cetmix_tower.py

```diff
@@ -1,6 +1,9 @@
 # Copyright (C) 2024 Cetmix OÜ
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import paramiko
```

Comment:
There is already an [SSH class](https://github.com/cetmix/cetmix-tower/blob/75ef96fc0a4224968b07ba30bf3a690d00598ee6/cetmix_tower_server/models/cx_tower_server.py#L40). You should use it instead of redefining everything once again.